Assignment/Conversion.c

```diff
@@ -8,26 +8,32 @@
 
 #pragma warning(disable:4996)
 #include <stdio.h>
+
 void DecToBinary (int n);
 void DecToHexa (int n);
-
-// Function to convert the decimal number to binary
+int bitLength;
+/// <summary>Decimal number to binary Conversion</summary>
 void DecToBinary (int n) {
-   int binary[32];
-   int i = 0;
-   int flag = 0; // Flag is to check if the number is negative
-   if (n == 0)    printf ("binary is 0");
+   int binary[32], i = 0;
+   int isNegative = 0; // Flag is to check if the number is negative
+   if (n == 0)   printf ("0");  
    if (n < 0) {
       n = -n;
-      flag = 1;
+      isNegative = 1;
    }
    while (n > 0) {
       binary[i] = n % 2;
-      n = n / 2;
+      n /= 2;
       i++;
    }
    // If the number was negative,apply two's complement
-   if (flag) {
+   if (isNegative) {
+      if (i <= 7)   bitLength = 8;
+      else   bitLength = 16;
+      for (int j = i; j < bitLength; j++) {
+         binary[j] = 0;
+      }

```

Comment:
Single line if and for statements does not require brackets. Make this change wherever applicable.